src/Summoner/Question/Data.hs

```diff
@@ -0,0 +1,50 @@
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Summoner.Question.Data
+  ( YesNoPrompt(..)
+  , mkYesNoPrompt
+  , mkDefaultYesNoPrompt
+  ) where
+
+import Data.Text (Text)
+
+data YesNoPrompt = YesNoPrompt
+  { yesno_target :: Text
+  , yesno_prompt :: Text
+  }
+
+-- | Build a prompt
+--
+-- For example,
+--
+-- @
+-- YesNoPrompt
+--   { yesno_target = "Cabal"
+--   , yesno_prompt = "Do you want to add a cabal integration?"}
+-- @
+-- will generate a following prompt message to the user
+--
+-- @
+-- Do you want to add a cabal integration? [y]/n
+--  -> y
+-- [Cabal] will be added to the project
+-- @
+mkYesNoPrompt ::
+     Text -- ^ Prompt Text
+  -> Text -- ^ Target Name
+  -> YesNoPrompt
+mkYesNoPrompt yesno_target yesno_prompt = YesNoPrompt {..}
```

Comment:
I don't see the point in this function, it's just a constructor, but not as safe as writing

```haskell
YesNoPrompt
    { yesNoTarget = "Cabal"
    , yesNoPrompt = "Do you want to add a cabal integration?"
    }
```

So I propose just remove this function. It's not even shorter to use `mkYesNoPrompt smth smth` instead of `YesNoPrompt smth smth` 😄